### DISCUSSION TITLE
Vite templates ready to test

### DISCUSSION THREAD

**Discussion by thednp** [vanjs-org/van#390] [Show and tell]
Hello there @Tao-VanJS and everyone else!

I'm pleased today to give you a sweet update: :biting_lip:  I've completed 3 fully working vite templates I'm sure you gonna love:
* [vite-starter-vanjs-ssr-jsx](https://github.com/thednp/vite-starter-vanjs-ssr-jsx) (express, vite, vanjs, van-plate) featuring SSR , SSG and JSX;
* [vite-starter-vanjs-ssr](https://github.com/thednp/vite-starter-vanjs-ssr) (express, vite, vanjs, van-plate) featuring SSR and SSG;
* [vite-starter-vanjs](https://github.com/thednp/vite-starter-vanjs) (vite, vanjs) featuring the most basic SPA setup.

These templates should put VanJS on the map. :smile:

Please clone, install, test and provide any feedback.

I've created a PR to include these to [create-vite-extra](https://github.com/bluwy/create-vite-extra/issues/66), I hope we get the green light.

Have a good one!

**Comment by thednp** [comment]
@kwameopareasiedu @b-rad-c @iuroc

I call on you to update your routers to work with SSR, you can make use of the above starter templates.

Have a good one!

**Reply by kwameopareasiedu** [reply]
@thednp Thanks very much for this. Will find time and look through...
